pandas/core/frame.py

```diff
@@ -7575,4 +7589,4 @@ def _from_nested_dict(data):
 
 
 def _put_str(s, space):
-    return ('%s' % s)[:space].ljust(space)
+    return '{s}'.format(s=s)[:space].ljust(space)
```

Comment:
To clarify, not at my laptop atm but I have a feeling just making this:
```python
u'{s}'.format(s=s)[:space].ljust(space)
```
would fix it, but it doesn't seem consistent with other .format occurrences, this could all be from a lack of understanding of how Pandas generally handles unicode though.

Reply:
The `u` prefix is appropriate here.